build.gradle

```diff
@@ -1,23 +1,13 @@
-// Gradle repositories and dependencies
-buildscript {
-    repositories {
-        jcenter()
-    }
-    dependencies {
-        classpath 'nl.javadude.gradle.plugins:license-gradle-plugin:0.11.0'
-        classpath 'com.github.jengelman.gradle.plugins:shadow:6.1.0'
-    }
+plugins {
+    id 'java'
+    id 'maven-publish'
+    id 'eclipse'
+    id 'idea'
+    id 'checkstyle'
+    id 'com.github.johnrengelman.shadow' version '7.0.0'
+    id "com.github.hierynomus.license" version "0.16.1"
```

Comment:
What's the problem with licenser? I recall licenser was made because this plugin ships a lot of unnecessary stuff and is a wrapper of a maven license fixing plugin

Reply:
Mixin uses this, so we should do to.

Reply:
Except it doesn't? It uses a third one https://github.com/SpongePowered/Mixin/blob/2046b44ad2fed1255256699f3b0b729bdb674369/build.gradle#L7

Reply:
Its the same one, the group is diffrent when using the gradle plugin block. See the readme here: https://github.com/hierynomus/license-gradle-plugin